src/settingsmanager.h

```diff
@@ -22,6 +22,11 @@ class SettingsManager : public QObject
     bool settingsExists(const QString &key);
     QVariant getSettings(const QString &key);
     qreal getZoomFactorByZimId(const QString &id);
+    int getKiwixServerPort() const { return m_kiwixServerPort; }
+    QString getKiwixServerIpAddress() const { return m_kiwixServerIpAddress; }
+    qreal getZoomFactor() const { return m_zoomFactor; }
+    QString getDownloadDir() const { return m_downloadDir; }
+    QString getMonitorDir() const { return m_monitorDir; }
```

Comment:
This `const`-ification of `getXXX()` methods should be in a commit of its own (preferably as the first commit of this PR so that new accessors are directly added as `const` methods in the next commit).